templates/dev.rake

```diff
@@ -1,4 +1,4 @@
-if Rails.env.development? || Rails.env.test?
+unless Rails.env.production?
```

Comment:
https://github.com/thoughtbot/suspenders/pull/860 to finish cleaning up staging references